facebook/__init__.py

```diff
@@ -335,13 +334,14 @@ def __init__(self, result):
             # OAuth 2.0 Draft 00
             try:
                 self.message = result["error"]["message"]
+                if not self.type:
+                    self.type = result["error"].get('type', '')
```

Comment:
I think these two lines are the only ones that need to be included in the pull request. Can you create a new pull request with them?